casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -233,9 +233,10 @@ class RuntimeManager private (val emptyStateHash: ByteString, runtimeContainer:
 object RuntimeManager {
   type StateHash = ByteString
 
-  def fromRuntime(active: Runtime): RuntimeManager = {
+  def fromRuntime(active: Runtime)(implicit scheduler: Scheduler): RuntimeManager = {
     active.space.clear()
     active.replaySpace.clear()
+    active.injectEmptyRegistryRoot[Task].unsafeRunSync
```

Comment:
I feel like this should be in Genesis.scala
You don't want to do this every time you create a Runtime, only when you create the Genesis block.

Reply:
I disagree. I can't think of any time that we would want a `RuntimeManager`, but not want to be able to use the registry. This is just part of the setup process now; we clear everything, initialize the registry and then snapshot that state so that we can go back to an "empty" tuplespace whenever we need to.

Reply:
I had the same concern as @guardbotmk3 . It doesn't feel right that initialization of the Registry should be in `RuntimeManager`.